plugins/richcombo/plugin.js

```diff
@@ -45,6 +45,7 @@ CKEDITOR.plugins.add( 'richcombo', {
 		' onfocus="return CKEDITOR.tools.callFunction({focusFn},event);"' +
 		' onclick="' + specialClickHandler + 'CKEDITOR.tools.callFunction({clickFn},this);return false;">' +
 			'<span id="{id}_text" class="cke_combo_text cke_combo_inlinelabel">{label}</span>' +
+			'<span id="{id}_value_label" class="cke_combo_label"></span>' +
```

Comment:
I suppose there is a reason why the currently selected value needs to be in a separate label? Is it some a11y `good practice`? If not, maybe just updating the initial label element could be a better idea? It takes less code in a template.

Reply:
Actually the first idea was to add `_text` element as a second element referenced by `[aria-labelledby]` – as it would be much easier than introducing logic for changing `_label` element. Unfortunately, `_text` element contains the initial label if user does not select anything. Due to that I introduced `_value_label` element, creating even the more convoluted solution than the logic for changing `_label` element :see\_no\_evil:.

However I'm thinking about even different approach – using both `[aria-labelledby]` and `[aria-describedby]` to [cover more assistive technology than only screen readers](https://www.sarasoueidan.com/blog/accessible-text-labels/). Yet it needs some testing.

Reply:
I assume we stay with `[aria-labelledby]` only :)